DSCResources/MSFT_xADDomainController/MSFT_xADDomainController.psm1

```diff
@@ -79,24 +96,41 @@ function Get-TargetResource
 
 function Set-TargetResource
 {
+    [CmdletBinding()]
     param
     (
-        [Parameter(Mandatory)]
+        [Parameter(Mandatory = $true)]
+        [System.String]
```

Comment:
This is specified twice on this parameter

Reply:
I believe this is saying that line was removed and the other was added.

Reply:
You have System.String and String on the same parameter: